packages/ui/components/ui/tx/view/isc20-withdrawal.tsx

```diff
@@ -0,0 +1,51 @@
+import { Ics20Withdrawal } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/ibc/v1/ibc_pb';
+import { ViewBox } from './viewbox';
+import { ActionDetails } from './action-details';
+import { joinLoHiAmount } from '@penumbra-zone/types/src/amount';
+import { bech32Address } from '@penumbra-zone/bech32';
+
+export const Ics20WithdrawalComponent = ({ value }: { value: Ics20Withdrawal }) => {
```

Comment:
Ics20Withdraw's do not have a special private/public case fyi

<img width="908" alt="Screenshot 2024-03-19 at 3 52 15 PM" src="https://github.com/penumbra-zone/web/assets/16624263/6b51154c-aca9-495f-8046-6cb032c11539">

Reply:
Not immediately but we probably do want to have a view for the Ics20Withdrawal so that we can insert:

- token metadata
- address view (of return address)